patterns/home.php

```diff
@@ -180,19 +180,19 @@
 <!-- /wp:heading --></div>
 <!-- /wp:group -->
 
-<!-- wp:group {"style":{"spacing":{"blockGap":"var:preset|spacing|30"}},"layout":{"type":"flex","orientation":"vertical"}} -->
-<div class="wp-block-group"><!-- wp:paragraph -->
-<p><strong><?php echo esc_html__( '✓', 'twentytwentyfour' ); ?></strong>&nbsp;<?php echo esc_html__( 'Collaborate with fellow architects', 'twentytwentyfour' ); ?></p>
-<!-- /wp:paragraph -->
-
-<!-- wp:paragraph -->
-<p><strong><?php echo esc_html__( '✓', 'twentytwentyfour' ); ?></strong>&nbsp;<?php echo esc_html__( 'Showcase your projects', 'twentytwentyfour' ); ?></p>
-<!-- /wp:paragraph -->
-
-<!-- wp:paragraph -->
-<p><strong><?php echo esc_html__( '✓', 'twentytwentyfour' ); ?></strong>&nbsp;<?php echo esc_html__( 'Experience the world of architecture like never before', 'twentytwentyfour' ); ?></p>
-<!-- /wp:paragraph --></div>
-<!-- /wp:group --></div>
+<!-- wp:list {"style":{"spacing":{"padding":{"right":"0","left":"var:preset|spacing|10"},"margin":{"top":"var:preset|spacing|30","bottom":"var:preset|spacing|30"}}},"className":"is-style-checkmark-list"} -->
+<ul class="is-style-checkmark-list" style="margin-top:var(--wp--preset--spacing--30);margin-bottom:var(--wp--preset--spacing--30);padding-right:0;padding-left:var(--wp--preset--spacing--10)"><!-- wp:list-item -->
+<li>Collaborate with fellow architects</li>
+<!-- /wp:list-item -->
+
+<!-- wp:list-item -->
+<li>Showcase your projects</li>
+<!-- /wp:list-item -->
+
```

Comment:
This section should be replaced with the matching pattern. We can do that for all patterns afterward as a separate PR, or it can be fixed here.